### PR TITLE
Added button and card variants

### PR DESCRIPTION
**Pull request by Nagashree01**
1.Added Button variants
- Primary
- Link
- Icon Button
- Floating Action Butto
Button

2.Card Variants
- Cards with badgeswith shadow
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards

card1
card2
card3
card4

card5

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    border: 2px solid var(--black-color);
+    border-radius: 0.2rem;
+    position: relative;
+}
+
+.card-vertical{
+    display: flex;
+    flex-direction: column;
+    width:18rem;
+
```
Comment (JantuDeb) [inline components/card.css:14]: You can add a `gap:` here so there will be a space between image and text

**Review by JantuDeb — APPROVED** [review]
Looks good.

File: components/card.css (hunk 1 above)
Comment (Nagashree01) [reply components/card.css:15]: Yes I did this. It looks better now. Thank to you!

**Review by kotalwar-krishna — APPROVED** [review]
Looks Good

**Pull request merged**